source/content/guides/drupal/drupal-hosted-createbt/05-upgrade.md

```diff
@@ -28,7 +28,7 @@ integration: [--]
       -W --no-update
    ```
 
-1. The `pantheon-systems/drupal-integrations` project now includes a patch that backports a bugfix from Drupal 9 to Drupal 8 to display the correct version of your MariaDB server. If this patch is not installed, then your database version will always be reported as `MySQL 5.5.30`.
+1. The `pantheon-systems/drupal-integrations` project now includes a patch that backports a bugfix to display the correct version of your MariaDB server. If this patch is not installed, then your database version will always be reported as `MySQL 5.5.30`.
```

Comment:
This patch is for Drupal 8 only. Drupal 9.1 and later already correctly display the correct MariaDB server version.

Reply:
reverted changes